### ISSUE TITLE
Platform Backups: Backup /restore for MongoDB (Clinical)

### ISSUE THREAD

**Issue by akachru-github** [icgc-argo/roadmap#367]
- Using the template "Service Backup" Helm chart developed by Henrich, modify the chart to support MongoDB services. 
- after the blocking helm chart is done, copy pasta it and adjust as needed.
- test backups are done as needed.
- This ticket includes testing and documenting the backup and restore process. 
---
Helm chart for etcd backups used as a template:
https://github.com/icgc-argo/kube-infra/tree/master/etcd-backup
Installation documentation:
https://wiki.oicr.on.ca/pages/viewpage.action?spaceKey=icgcargotech&title=ETCD+backups

--
Output is 2 charts (one for clinical and one for lectern)

**Comment by hlminh2000** [comment]
lectern not complete due to not using vault, blocked by https://github.com/icgc-argo/roadmap/issues/395
created new ticket: https://github.com/icgc-argo/roadmap/issues/409
Left link to doc in: https://wiki.oicr.on.ca/display/icgcargotech/Clinical+mongo+backup

**Comment by rosibaj** [comment]
@hlminh2000 is there anything we need to do specifically to deploy this to prod? is it just a chart version update?

**Comment by hlminh2000** [comment]
we need to install the chart, once I've actually fixed the problem mentioned in demo

**Comment by rosibaj** [comment]
@hlminh2000 can i get a link to the chart to install on this ticket to add to realease management rundown?

**Comment by hlminh2000** [comment]
@rosibaj here's the chart repo: https://github.com/icgc-argo/kube-infra/tree/master/clinical-mongo-backup

**Comment by rosibaj** [comment]
@andricDu to confirm, this is deployed to production?  if no response, will close ticket.
